### PR TITLE
[FIX] mail: avoid star flicker when hovering a message

### PR DESCRIPTION
**Pull request by phenix-factory**
task-2278441

### CODE REVIEW

File: addons/mail/static/src/components/message/message.scss [hunk 1]
```diff
@@ -175,11 +175,11 @@
     &:hover, &.o-clicked {
 
         .o_Message_command {
-            display: flex;
+            opacity: 1;
         }
 
         .o_Message_commands {
-            display: flex;
+            opacity: 1;
```
Comment (seb-odoo) [inline addons/mail/static/src/components/message/message.scss:182]: don't change this one, it was flex for display reasons I think, only the one that has opacity 0 should have opacity 1

Comment (alexkuhn) [reply addons/mail/static/src/components/message/message.scss:178]: All `display: flex` in style were alternatives to `opacity`, so they are safe for removal

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

The scss of message component is a bad example no embracing hover concern in template, which results in scss hierachy and increased specificy. To a lesser extend, same problem with `o-clicked` not being propagated to dom child nodes that need it.

**Comment by alexkuhn** [comment]
FYI task-2338021 for cleaning this in `Message` component.
https://www.odoo.com/web#id=2338021&action=3206&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720

**Pull request closed without merging**